### ISSUE TITLE
mergeTrees error on namespaced packages

### ISSUE THREAD

**Issue by karellm** [asakusuma/ember-rollup#19]
I’m working on an addon for ckeditor 5 that publishes ES6 modules. I do something like:

```
var emberRollup = require('ember-rollup');
var runtimeDependencies = [
  '@ckeditor/ckeditor5-editor-balloon',
  '@ckeditor/ckeditor5-essentials',
  '@ckeditor/ckeditor5-paragraph',
  '@ckeditor/ckeditor5-basic-styles'
];

module.exports = emberRollup(runtimeDependencies, {
  name: 'ember-ckeditor-5',
  isDevelopingAddon: function() {
    return true;
  }
});
```

But then I get an error: 

```
Error: Merge error: file @ckeditor/CHANGELOG.md exists in /.../broccoli_merge_trees-input_base_path-JFtNWam8.tmp/0 and /.../broccoli_merge_trees-input_base_path-JFtNWam8.tmp/1

Pass option { overwrite: true } to mergeTrees in order to have the latter file win.
```

Does it ring a bell? Thanks for your help

**Comment by karellm** [comment]
I identified the error as coming from: https://github.com/asakusuma/ember-rollup/blob/master/src/rollup-tree.js#L50

```js
      var moduleDir = path.dirname(dep.moduleName);
      if (dep.moduleName.startsWith('@')) {
        moduleDir = dep.moduleName;
      }
```

This look like it does the trick. @asakusuma do you want a PR?

**Comment by asakusuma** [comment]
@karellm this should be fixed now as of `ember-rollup@2.0.0`
